src/server/session.ts

```diff
@@ -1178,7 +1178,7 @@ namespace ts.server {
         private getRenameInfo(args: protocol.FileLocationRequestArgs): RenameInfo {
             const { file, project } = this.getFileAndProject(args);
             const position = this.getPositionInFile(args, file);
-            const preferences = this.getHostPreferences();
+            const preferences = { ...this.getHostPreferences(), ...this.getPreferences(file) };

```

Comment:
Is there a reason we shouldn't just change 
```ts
        getPreferences(file: NormalizedPath): protocol.UserPreferences {
            const info = this.getScriptInfoForNormalizedPath(file);
            return info && info.getPreferences() || this.hostConfiguration.preferences;
        }
```
in `editorServices.ts` into
```ts
        getPreferences(file: NormalizedPath): protocol.UserPreferences {
            const info = this.getScriptInfoForNormalizedPath(file);
            return {...info && info.getPreferences(), ...this.hostConfiguration.preferences};
        }
```
?

Reply:
Good idea. Changed to that.